.github/workflows/smoke.yml

```diff
@@ -28,6 +28,7 @@ jobs:
         shell: bash
         env:
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          ANTHROPIC_API_KEY: ${{ secrets.ANTHROPIC_API_KEY }}
```

Comment:
Here we add the anthropic key to the env. But notice, in conftest.py (below) there is logic that conditionally _removes_ the unneeded key based on the label of the test. In this way we don't accidentally use the default key.